File: lection04/app_java/src/main/java/ru/hse/lection04/businesslayer/Channels.java
package ru.hse.lection04.businesslayer;

import ru.hse.lection04.R;

/**
 * Каналы, которые есть в приложении
 */
public enum Channels {
    CONNECTIVITY("CONNECTIVITY", R.string.app_channel_connectivity);


    public final String id;
    public final int nameResId;


    Channels(String id, int nameResId) {
        this.id = id;
        this.nameResId = nameResId;
    }
}

File: lection04/app_kotlin/src/main/java/ru/hse/lection04/businesslayer/Channels.kt
package ru.hse.lection04.businesslayer

import ru.hse.lection04.R

/**
 * Каналы, которые есть в приложении
 */
enum class Channels(val id: String, val nameResId: Int) {
    CONNECTIVITY("CONNECTIVITY", R.string.app_channel_connectivity);
}